website/content/en/docs/building-operators/ansible/reference/scaffolding.md

```diff
@@ -21,28 +21,20 @@ table describes a basic rundown of each generated file/directory.
 | File/Folders   | Purpose |
 | :---           | :---    |
 | Dockerfile | The Dockerfile for building the container image for the operator. |
-| Makefile | Contains make targets for building, publishing, deploying the
-container image that wraps the operator binary, and make targets for
-installing and uninstalling the CRD. |
+| Makefile | Contains make targets for building, publishing, deploying the container image that wraps the operator binary, and make targets for installing and uninstalling the CRD. |
```

Comment:
Could we not say the same but in a less verbose way? E.g `Contains the targets used to manage the project.`

Reply:
I prefer to leave this as-is for now. All I changed was the table formatting.